tests/functional/platforms/02-host-to-platform-upgrade/reference.log

```diff
@@ -1,6 +1,10 @@
 Initial point: 1
 Final point: 1
-[no_settings.1] -triggered off []
-[t1.1] -triggered off []
-[t2.1] -triggered off []
-[fin.1] -triggered off ['no_settings.1', 't1.1', 't2.1']
+[fin.20200101T0000Z] -triggered off ['no_settings.20200101T0000Z', 't1.20200101T0000Z', 't2.20200101T0000Z']
+[fin.20210101T0000Z] -triggered off ['no_settings.20210101T0000Z', 't1.20210101T0000Z', 't2.20210101T0000Z']
+[no_settings.20200101T0000Z] -triggered off []
+[no_settings.20210101T0000Z] -triggered off []
+[t1.20200101T0000Z] -triggered off []
+[t1.20210101T0000Z] -triggered off []
+[t2.20200101T0000Z] -triggered off []
+[t2.20210101T0000Z] -triggered off []
```

Comment:
It's easier to compare cycle points when the tests fail if using integer cycling rather than datetime